### PR TITLE
Minor cleanup/simplification, no functional changes

### PR DESCRIPTION
**Pull request by hmusum**

### CODE REVIEW

File: config-model/src/main/java/com/yahoo/vespa/model/content/StorageGroup.java [hunk 1]
```diff
@@ -412,21 +412,17 @@ else if (! nodesElement.isPresent() && subGroups.isEmpty() && context.getDeployS
         }
 
         private Optional<String> childAsString(Optional<ModelElement> element, String childTagName) {
-            if ( ! element.isPresent()) return Optional.empty();
-            return Optional.ofNullable(element.get().childAsString(childTagName));
+            return element.map(modelElement -> modelElement.childAsString(childTagName));
```
Comment (jonmv) [inline config-model/src/main/java/com/yahoo/vespa/model/content/StorageGroup.java]: `element.get().childAsString(childTagName)` is never `null`?

**Review by jonmv — APPROVED** [review]
One question :)

File: config-model/src/main/java/com/yahoo/vespa/model/content/StorageGroup.java (hunk 1 above)
Comment (hmusum) [reply config-model/src/main/java/com/yahoo/vespa/model/content/StorageGroup.java]: Hm, nice catch. Shouldn't always trust the IDE's suggested changes it seems. Will fix.

Comment (jonmv) [reply config-model/src/main/java/com/yahoo/vespa/model/content/StorageGroup.java]: Hmm, `.map(...)` handles the result being null. I don't think it did before?

Comment (jonmv) [reply config-model/src/main/java/com/yahoo/vespa/model/content/StorageGroup.java]: Bah, seems it always did :p

Comment (jonmv) [reply config-model/src/main/java/com/yahoo/vespa/model/content/StorageGroup.java]: Well, learned something new today, then :)

**Pull request merged**